includes/User.php

```diff
@@ -4218,6 +4218,20 @@ public static function crypt( $password, $salt = false ) {
 	 * @return Boolean
 	 */
 	public static function comparePasswords( $hash, $password, $userId = false ) {
+		// Wikia change - begin
+		// @see PLATFORM-2502 comparing new passwords in PHP code.
+		// @todo mech remove after the new password hashing is implemented.
+		$caller = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS)[1];
+		Wikia\Logger\WikiaLogger::instance()->debug(
```

Comment:
Alright. Let us be weary if we notice some unexpected spike in generated logs after this goes live.